controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2324 @@
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5250
+  - id: 3.4.3.8
+    title: Ensure nftables rules are permanent (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5252
+  - id: 3.4.4.1.1
+    title: Ensure iptables default deny firewall policy (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5253
+  - id: 3.4.4.1.2
+    title: Ensure iptables loopback traffic is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 3.4.4.1.3
+    title: Ensure iptables outbound and established connections are configured (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5255
+  - id: 3.4.4.1.4
+    title: Ensure iptables firewall rules exist for all open ports (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7190
+  - id: 3.4.4.1.5
+    title: Ensure iptables is enabled and active (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5257
+  - id: 3.4.4.2.1
+    title: Ensure ip6tables default deny firewall policy (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5258
+  - id: 3.4.4.2.2
+    title: Ensure ip6tables loopback traffic is configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 3.4.4.2.3
+    title: Ensure ip6tables outbound and established connections are configured (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7191
+  - id: 3.4.4.2.4
+    title: Ensure ip6tables firewall rules exist for all open ports (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/7192
+  - id: 3.4.4.2.5
+    title: Ensure ip6tables is enabled and active (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 3.5
+    title: Ensure wireless interfaces are disabled (Automated)
+    levels:
+      - l1_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - wireless_disable_interfaces
+
+  - id: 3.6
+    title: Disable IPv6 (Manual)
+    levels:
+      - l2_server
+      - l2_workstation
+    automated: yes
```

Comment:
Hello, I incidentally stumbled upon this. This needs to be fixed. If the rule is marked as manual, we do not automate it. Moreover, this rule no longer works in rhel7 and later, so I wouldn't put it even into related_rules.

Reply:
Well spotted. I've fixed this one and gone through the file with grep to check for any other instances of controls which are marked as Manual but which we have still set to `automated: yes`. There were none other than this one.